src/Bravo3/Orm/Services/Io/Reader.php

```diff
@@ -56,9 +57,39 @@ public function getPropertyValue($name)
         } elseif ($relationship = $this->metadata->getRelationshipByName($name)) {
             $getter = $relationship->getGetter();
             return $this->entity->$getter();
-        } else {
-            throw new InvalidArgumentException("No column/relationship found for property '".$name."'");
```

Comment:
Yeah, I had some issues here. Initially I had the call to `getRelationColumnValue()` in the `getRelationColumnValue()` method something like:

```
    public function getPropertyValue($name)
    {
        if ($column = $this->metadata->getColumnByProperty($name)) {
            $getter = $column->getGetter();
            return $this->entity->$getter();
        } elseif ($relationship = $this->metadata->getRelationshipByName($name)) {
            $getter = $relationship->getGetter();
            return $this->entity->$getter();
        } elseif ($value = $this->getRelationColumnValue($name)) {
            $values[] = $value;
        } else {
            throw new InvalidArgumentException("No column/relationship found for property '".$name."'");
        }
```

Not sure why, but this would always through the exception.